README.md

```diff
@@ -241,10 +250,10 @@ are banned from playing unranked game
 byoyomi,simple,canadian,absolute,none)
 
   ```--minmaintime```  or ```--0mt``` Minimum seconds of main time (rejects time 
-control simple and none)
+control simple and none) (default 60)
 
   ```--maxmaintime```  or ```--1mt``` Maximum seconds of main time (rejects time 
-control simple and none)
+control simple and none) (default 1800)
```

Comment:
This is only 30 minutes. How about making default max main time 2 or 4 hours. The default on max time is more of a "prevent unintended unlimited time". If people want less than a high amount they can lower in settings. 30 minutes feels too low for default max, since often people do in fact play bots with longer clocks?

Reply:
i see, this is why i wanted to make separate time settings for everygame speed : 
- `--maxmaintimeblitzlive 60*60`
- `--maxmaintimecorr 60*60*24*7`

i think every speed should have its own time speed @roy7 
more than 1 hour main time is not a live game anymore (blitz and live are kidna same so same category)
if main time is more than 1 hour, i suggest the human player should play a corr game instead

as you said, bot admin is free to increase to `--maxmaintimeblitzlive 60*60*4` if want @roy7 
i think this makes more sense 

see #125 
also, with a config file we'd be able to do math like that automatically which is also a nice advantage

to sum up, i can increase to 1 hour, but more than that should be a corr game by default, unless bot admin wants to manually change that
(after main time there are still periods : default is 1 hour + 20 x 2 min if i remember correctly, theres plenty of time for a live game, much more than on foxwq or tygem for example)

Reply:
I agree with Roy here. Default should be something large like 2-4 hours. If bot admins find that too much, they can decrease it.

Reply:
ok, i'll increase it to 2 hours then, but remember that there is still periodtime on top of that

and with maxmaintime 2 hours, it means corr games are not possible, all the more reason to add a specific setting for blitz, live corr, see my messages in https://github.com/online-go/gtp2ogs/pull/123#issuecomment-462112142